gsa/src/web/graphql/wizards.js

```diff
@@ -0,0 +1,77 @@
+/* Copyright (C) 2020 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation, either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+import {useCallback} from 'react';
+
+import {useMutation} from '@apollo/client';
+
+import {dateTimeWithTimeZone} from 'gmp/locale/date';
+
+import {CREATE_TARGET} from 'web/graphql/targets';
+
+import {CREATE_TASK, START_TASK} from 'web/graphql/tasks';
+
+import useUserSessionTimeout from 'web/utils/useUserSessionTimeout';
+import useUserTimezone from 'web/utils/useUserTimezone';
+
+export const useRunQuickFirstScan = () => {
+  const [sessionTimeout] = useUserSessionTimeout();
+  const [userTimezone] = useUserTimezone();
+
+  const [createTarget] = useMutation(CREATE_TARGET);
+  const [createTask] = useMutation(CREATE_TASK);
+  const [startTask] = useMutation(START_TASK);
```

Comment:
didn't we implement custom hooks for these mutations already?

Reply:
We do. And now I have zero clue why I didn't use them already.